seed_services_cli/stage_based_messaging.py

```diff
@@ -167,21 +167,66 @@ def messages_import(ctx, csv, json):
             api.create_message(msg)
 
 
+@click.option(
+    '--csv', type=click.File(file_open_mode),
+    help=('CSV file with columns for the endpoint'))
+@click.option(
+    '--json', type=click.File(file_open_mode),
+    help=('JSON objects, one per line for the endpoint'))
+@click.pass_context
+def messages_update(ctx, csv, json):
+    """ Update messages in the Stage Based Messaging service.
+        binary_content fields should refer to filename in the current folder
+    """
+    if not any((csv, json)):
+        raise click.UsageError("Please specify either --csv or --json.")
+    api = get_api_client(ctx.obj.stage_based_messaging.api_url,
+                         ctx.obj.stage_based_messaging.token)
+    if csv:
+        messages = messages_from_csv(csv)
+    elif json:
+        messages = messages_from_json(json)
+    params = {}
+    for message in messages:
+        params["messageset"] = message["messageset"]
+        params["lang"] = message["lang"]
+        params["sequence_number"] = message["seqno"]
+        results = list(api.get_messages(params=params)['results'])
+        number_of_results = len(results)
+        if number_of_results == 1:
+            message["id"] = results[0]["id"]
+        elif number_of_results < 1:
+            raise click.UsageError("Multiple messages with messageset: %s, language: %s, \
+            sequence number: %s found.") % \
+                (message["messageset"], message["lang"], message["seqno"])
+        elif number_of_results > 1:
+            raise click.UsageError("Message with messageset: %s, language: %s, \
+            sequence number: %s not found.") % \
+                (message["messageset"], message["lang"], message["seqno"])
+    for message in messages:
+        if message["binary_content"] is not None and \
+                message["binary_content"] != "":
+            message = create_binarycontent(api, message)
+        click.echo("Updating message to messageset %(messageset)s." %
+                   message)
+        api.update_message(message)
```

Comment:
Over here you will need to send the message['id'] to the `update_message` function.